tests/e2e/pageobjects/ide/CheCodeLocatorLoader.ts

```diff
@@ -40,6 +40,9 @@ export class CheCodeLocatorLoader extends LocatorLoader {
 				WelcomeContent: {
 					text: By.xpath('//*[@class="dialog-message-text" and contains(text(), "trust")]'),
 					button: By.xpath('//div[@class="monaco-dialog-box"]//a[@class="monaco-button monaco-text-button"]')
+				},
+				ScmView: {
+					actionConstructor: (title: string) => By.xpath(`.//a[@title='${title}']`)
```

Comment:
Why do we reassign it? I don`t see any difference with default value
https://github.com/redhat-developer/vscode-extension-tester/blob/038b2fc40f01593278e51d03fdbe95e12b32984b/locators/lib/1.37.0.ts#L338

Reply:
It seems, it was redefined in 1.70.js -> 

```
ScmView: {
            action: monaco_page_objects_1.By.className('action-item menu-entry'),
            actionConstructor: (title) => monaco_page_objects_1.By.xpath(`.//li[@title='${title}']`)
        },
```
And I provided the error about it above, in the description

Reply:
thank you for explanations